### PR TITLE
Relationship decomposition

### PR DESCRIPTION
**Pull request by jkoenig134**
# Readiness checklist

-   [ ] I added/updated tests.
-   [x] I ensured that the PR title is good enough for the changelog.
-   [x] I labeled the PR.

### CODE REVIEW

**Review by sebbi08 — APPROVED** [review]

File: src/modules/coreHttpApi/openapi.yml [hunk 1]
```diff
@@ -1039,9 +1039,8 @@ paths:
           schema:
             $ref: "#/components/schemas/AttributeID"
       responses:
-        200:
-          description: |
-            Success. Deleting an repository attribute.
+        204:
+          description: Success. Deleting an repository attribute.
```
Comment (britsta) [inline src/modules/coreHttpApi/openapi.yml:1043]: Deleting **a** RepositoryAttribute.

File: test/lib/validation.ts [hunk 2]
```diff
@@ -47,6 +48,7 @@ export function validateSchema(schemaName: ValidationSchema, obj: any): void {
 
 const schemaDefinition = getJSONSchemaDefinition();
 const ajvInstance = new ajv({ schemas: [schemaDefinition], allowUnionTypes: true });
+type ExtendetError = ErrorObject<string, Record<string, any>, unknown> & { value: any };
```
Comment (britsta) [inline test/lib/validation.ts:51]: "type Extende**d**Error" instead of "type ExtendetError".

File: src/modules/coreHttpApi/openapi.yml (hunk 1 above)
Comment (jkoenig134) [reply src/modules/coreHttpApi/openapi.yml]: yeah, I saw this too but was too lazy :D

File: test/lib/validation.ts [hunk 3]
```diff
@@ -66,9 +68,32 @@ expect.extend({
...
+
+        if (!actual.isSuccess) {
+            const message = `expected a successful result; got an error result with the error message '${actual.error.message}'.`;
+            return { pass: false, message: () => message };
+        }
+
+        if (typeof actual.result !== "undefined") {
+            return {
+                pass: false,
+                message: () => `expected a successful result to to be a void result, but got the following result: ${JSON.stringify(actual.result, null, 2)}`
```
Comment (britsta) [inline test/lib/validation.ts:96]: Last comment, rest LGTM.

"Expected a successful result to ~~to~~ be a void result".

**Review by britsta — APPROVED** [review]
🐸

**Pull request merged**